packages/card/person-card/index.tsx

```diff
@@ -18,13 +18,13 @@ interface PersonCardProps {
 function Icon({ url }) {
   let icon
   if (url.includes('twitter')) {
-    icon = <IconTwitter16 />
+    icon = <IconTwitter16 data-testid={'wpl-personcard-twitter-icon'} />
   }
   if (url.includes('github')) {
-    icon = <IconGithub16 />
+    icon = <IconGithub16 data-testid={'wpl-personcard-github-icon'} />
   }
   if (url.includes('linkedin')) {
-    icon = <IconLinkedin16 />
+    icon = <IconLinkedin16 data-testid={'wpl-personcard-linkedin-icon'} />
```

Comment:
🤔 This feels...goofy to me? I was looking for a way to single out each of these icons in the test to ensure they each appear when the corresponding links are passed to the component. 

Perhaps I should rethink the test? 🤔

Reply:
Will address this in a later PR if necessary!